<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSelArcos</class>
 <widget class="QDialog" name="frmSelArcos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Seleccione los arcos que conforman el per√≠metro de la finca y pulse Aceptar: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttGrabarFinca">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GrabarFinca</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNumEltos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de elementos seleccionados: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCapa">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capa del 1er elto:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttGrabarLindes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GrabarLindes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttGrabarImagenes">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GrabarImagenes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttCambiaImg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
